lib/reporters/basic.js

```diff
@@ -3,9 +3,10 @@ const mapValues = require('lodash.mapvalues');
 
 function metrics (set) {
   const m = {
-    render: getProfileTime('Paint', set),
+    firstPaint: getProfileTime('Paint', set),
```

Comment:
Do you think first and last paint should go next to each other or is the a reason to keep them separate?